### PR TITLE
rel: Release notes and prep for 2.8

### PR DESCRIPTION
**Pull request by kentquirk**
## Which problem is this PR solving?

- Release notes for 2.8
- Changelog for 2.8
- Fix a few issues with attribution of previous changelog entries

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -2,6 +2,47 @@
...
+
+* The root span arrives
+* The TraceTimeout expires
+
+Release 2.8 introduces a new feature, `SpanLimit`, which provides a third way to cause Refinery to make a trace decision. It sets the maximum number of descendants that a trace can have before it gets marked for a trace decision. This is to help with the memory consumption due to very large traces.
+
+Suppose, for example, that a service generates a single trace with 10,000 spans. If SpanLimit is set to 1000, once the first 1000 spans have arrived, Refinery will immediately make a decision to keep or drop the trace. Every additional span is dispatched (using the same decision) without storing it. This means that Refinery never had to keep all 10,000 spans in its memory at one time.
+
+### Other Improvements
+
```
Comment (cartermp) [inline RELEASE_NOTES.md:41]:
```suggestion

* Refinery rules now allow specifying `root.` prefixes for fields in dynamic samplers.
```

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -37,6 +37,11 @@ Release 2.8 introduces a new feature, `SpanLimit`, which provides a third way to
 
 Suppose, for example, that a service generates a single trace with 10,000 spans. If SpanLimit is set to 1000, once the first 1000 spans have arrived, Refinery will immediately make a decision to keep or drop the trace. Every additional span is dispatched (using the same decision) without storing it. This means that Refinery never had to keep all 10,000 spans in its memory at one time.
 
+### `In` and `NotIn` Operators in Rules
```
Comment (cartermp) [inline RELEASE_NOTES.md:40]: There's a discrepancy in how we document here versus in the PR: https://github.com/honeycombio/refinery/pull/1302/files#diff-adf5ae21a5e738c1aab5407a6a4c83df85668d66363f27ccaea4788fba3e178bR672-R673

Comment (cartermp) [reply RELEASE_NOTES.md:40]: In the below text, and the rules-complete file, the operator is shown as `In`/`NotIn`. But in the yaml file I link to, it's shown as `in`/`not-in`. Which is the operator? Or is the rulesmeta file just an inside baseball type representation?

Comment (kentquirk) [reply RELEASE_NOTES.md:40]: excellent catch. Thank you. The operator is `in` and `not-in` and I will fix those bugs!

**Review by cartermp — APPROVED** [review]

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -2,6 +2,58 @@
 
 While [CHANGELOG.md](./CHANGELOG.md) contains detailed documentation and links to all the source code changes in a given release, this document is intended to be aimed at a more comprehensible version of the contents of the release from the point of view of users of Refinery.
 
+## Version 2.8.0
```
Comment (VinozzZ) [inline RELEASE_NOTES.md:5]: Do we also want to mention the layered config support in this release?

File: RELEASE_NOTES.md [hunk 4]
```diff
@@ -2,6 +2,61 @@
...
+
+This release has a several significant changes that should make Refinery easier to operate at scale.
```
Comment (akvanhar) [inline RELEASE_NOTES.md:7]:
```suggestion
This release has a several significant changes that makes Refinery easier to operate at scale.
```

**Review by VinozzZ — APPROVED** [review]

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,60 @@
...
+
+### Fixes
+- fix: periodically clean up recent_dropped_traces cache (#1312) | [Yingrong Zhao](https://github.com/vinozzZ)
+- fix: revert the revert -- that wasn't the problem (#1311) | [Kent Quirk](https://github.com/kentquirk)
+- fix: revert "Use HTTP/2 for all upstream and peer-to-peer connections… (#1310) | [Kent Quirk](https://github.com/kentquirk)
+- fix: join peer list only after refinery is ready to accept traffic (#1309) | [Yingrong Zhao](https://github.com/vinozzZ)
+- fix: use float histogram for otel metrics (#1303) | [Kent Quirk](https://github.com/kentquirk)
+- fix: escape use input in debug route (#1299) | [Tyler Helmuth](https://github.com/TylerHelmuth)
+- fix: use trace.DescendantCount for span limit (#1297) | [Yingrong Zhao](https://github.com/vinozzZ)
+- fix: support TLS connections to Redis (#1285) | [Yingrong Zhao](https://github.com/vinozzZ)
```
Comment (VinozzZ) [inline CHANGELOG.md:29]: There is a duplicate entry for this fix

**Review by VinozzZ — APPROVED** [review]

**Review by erjohnson — APPROVED** [review]

**Pull request merged**